targets/TARGET_NXP/TARGET_MCUXpresso_MCUS/TARGET_IMX/spi_api.c

```diff
@@ -124,16 +124,21 @@ int spi_master_write(spi_t *obj, int value)
 int spi_master_block_write(spi_t *obj, const char *tx_buffer, int tx_length,
```

Comment:
@artokin one file left with permissions change

Reply:
Thanks @0xc0170 , fixed in the next commit.